smrf/tests/smrf_test_case.py

```diff
@@ -121,11 +121,25 @@ def assert_gold_equal(gold, not_gold, error_msg):
         """
 
         if os.getenv('NOT_ON_GOLD_HOST') is None:
-            np.testing.assert_array_equal(
-                not_gold,
-                gold,
-                err_msg=error_msg
-            )
+            try:
+                np.testing.assert_array_equal(
+                    not_gold,
+                    gold,
+                    err_msg=error_msg
+                )
+            except AssertionError:
+                for rtol in [1e-5, 1e-4]:
+                    status = np.allclose(
+                        not_gold,
+                        gold,
+                        atol=0,
+                        rtol=rtol
+                    )
+                    if status:
+                        break
+                if not status:
+                    raise AssertionError(error_msg)
+                print('Arrays are not exact match but close with rtol={}'.format(rtol))  # noqa
```

Comment:
Is this status check needed?
Wouldn't it raise another `AssertionError` on the `allclose`?

Reply:
I moved away from the `np.testing.allclose`. Now that I'm looking at it again, I can use the testing version.

Reply:
Now I remember why! It may fail with the 1e-5 but not with the 1e-4. So I didn't want the test to fail until it made it all the way through.